src/multi_vector_simulator/B0_data_input_json.py

```diff
@@ -311,4 +326,12 @@ def load_json(
     # add default value if the field PATHS_TO_PLOTS is not already present
     if PATHS_TO_PLOTS not in dict_values:
         dict_values.update(copy.deepcopy(DICT_PLOTS))
+
+    # raise a warning if required parameters are missing, see REQUIRED_MVS_PARAMETERS in
+    # constants.py for more information, note json and csv required parameter are independent from
+    # one another at the moment
+    compare_input_parameters_with_reference(
```

Comment:
We should (in another PR) check if `utils.compare_input_parameters_with_reference` and `A1.check_for_official_extra_parameters` are so similar that they can be simplified.

Reply:
Agreed